File: presentation/src/main/java/com/github/anrimian/musicplayer/ui/settings/display/DisplaySettingsView.java
package com.github.anrimian.musicplayer.ui.settings.display;

import moxy.MvpView;
import moxy.viewstate.strategy.alias.AddToEndSingle;

public interface DisplaySettingsView extends MvpView {

    @AddToEndSingle
    void showCoversChecked(boolean checked);

    @AddToEndSingle
    void showCoversInNotificationChecked(boolean checked);

    @AddToEndSingle
    void showColoredNotificationChecked(boolean checked);

    @AddToEndSingle
    void showNotificationCoverStubChecked(boolean checked);

    @AddToEndSingle
    void showCoversOnLockScreenChecked(boolean checked);

    @AddToEndSingle
    void showCoversInNotificationEnabled(boolean enabled);

    @AddToEndSingle
    void showColoredNotificationEnabled(boolean enabled);

    @AddToEndSingle
    void showNotificationCoverStubEnabled(boolean enabled);

    @AddToEndSingle
    void showShowCoversOnLockScreenEnabled(boolean enabled);
}

File: app/src/main/java/com.github.anrimian.musicplayer/ui/settings/display/DisplaySettingsView.kt
package com.github.anrimian.musicplayer.ui.settings.display

import moxy.MvpView
import moxy.viewstate.strategy.alias.AddToEndSingle

interface DisplaySettingsView : MvpView {

    @AddToEndSingle
    fun showFileNameEnabled(enabled: Boolean)

    @AddToEndSingle
    fun showCoversChecked(checked: Boolean)

    @AddToEndSingle
    fun showCoversInNotificationChecked(checked: Boolean)

    @AddToEndSingle
    fun showColoredNotificationChecked(checked: Boolean)

    @AddToEndSingle
    fun showNotificationCoverStubChecked(checked: Boolean)

    @AddToEndSingle
    fun showCoversOnLockScreenChecked(checked: Boolean)

    @AddToEndSingle
    fun showCoversInNotificationEnabled(enabled: Boolean)

    @AddToEndSingle
    fun showColoredNotificationEnabled(enabled: Boolean)

    @AddToEndSingle
    fun showNotificationCoverStubEnabled(enabled: Boolean)

    @AddToEndSingle
    fun showCoversOnLockScreenEnabled(enabled: Boolean)

}